content/memos/_index.fr.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Mémos
+html_description: Des exemples proposés par des membres de la communauté de ce qu'Open Terms Archive permet de repérer.
```

Comment:
Can you use non breakable spaces in `Open Terms Archive` to avoid line break in the middle of the name